django/forms/jinja2/django/forms/widgets/select.html

```diff
@@ -0,0 +1,5 @@
+<select name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %}>{% for group_name, group_choices, group_index in widget.optgroups %}{% if group_name %}
```

Comment:
If you want to prevent whitespace in jinja2 you can use `{%- tag %}` or `{% tag -%}`.
Then you can use indentation in the templates without outputting the whitespace.

See also: http://jinja.pocoo.org/docs/dev/templates/#whitespace-control